.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Audio files for testing
+audio/
```

Comment:
Is this related?

Reply:
Oh not with the template. We just updated the .gitignore. I guess I could have committed this in a different PR mb.